types/types.go

```diff
@@ -410,13 +411,14 @@ type DefaultNetworkSettings struct {
 
 // MountPoint represents a mount point configuration inside the container.
 type MountPoint struct {
-	Name        string `json:",omitempty"`
+	Type        mount.Type `json:",omitempty"`
+	Name        string     `json:",omitempty"`
 	Source      string
 	Destination string
```

Comment:
Wish we could alias these fields. I'm guessing this is exported via docker info or docker inspect?